app/views/admin/creator_settings/_form.html.erb

```diff
@@ -8,20 +8,24 @@
 </div>
 
 <div class="crayons-field mt-6 align-left">
-  <%= label_tag :logo_svg, class: "crayons-field__label" do %>
+  <div class="crayons-field__label relative">
```

Comment:
This is no longer the label. There is now a label that looks like a button hiding the actual input of type file. It didn't make sense to make the old label a subheading, so I'm not sure what to use here from a semantic standpoint. Maybe you have a suggestion @aitchiss.

Reply:
This is how I approached the refactor to the ImageUploader button too, as like you say you can't style the file input button (maybe one day?!)

I would tend to agree that "Logo" probably doesn't need to be associated with the file input, since the label text already says "Upload logo" and is pretty self explanatory. We should link the description with the input though by using `aria-describedby="logo-subtitle"` on line 20, just to get that extra info passed along to screen reader users